CSharp/Library/Microsoft.Bot.Connector.Shared/MicrosoftAppCredentials.cs

```diff
@@ -83,7 +83,18 @@ public MicrosoftAppCredentials(IConfiguration configuration, ILogger logger = nu
         public string MicrosoftAppId { get; set; }
         public string MicrosoftAppPassword { get; set; }
 
-        public virtual string OAuthEndpoint { get { return JwtConfig.ToChannelFromBotLoginUrl; } }
+        public virtual string OAuthEndpoint
+        {
+            get
+            {
+                string tenant = null;
+#if NET45
+                // Advanced user only. TODO: add doc link.

```

Comment:
Please resolve the "todo".

Reply:
can't yet. don't have the doc.

Reply:
When will we have the doc? Seems like something you could write (a quick markdown file) and Kamran can integrate. If this at least pointed to an aka.ms link that Kamran manages, then we can update the doc later.

Reply:
added aka.ms link.